# This patch adds config support for recognising (and therefore using)
# version 4.3 of the BerkelyDB
--- cyrus-sasl-2.1.19/cmulocal/berkdb.m4.orig	2005-07-10 23:09:33.323172813 -0700
+++ cyrus-sasl-2.1.19/cmulocal/berkdb.m4	2005-07-10 23:09:49.866665096 -0700
@@ -211,7 +211,7 @@
 	    BDB_LIBADD=""
 	fi
 
-        for dbname in db-4.2 db4.2 db42 db-4.1 db4.1 db41 db-4.0 db4.0 db-4 db40 db4 db-3.3 db3.3 db33 db-3.2 db3.2 db32 db-3.1 db3.1 db31 db-3 db30 db3 db
+        for dbname in db-4.3 db4.3 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db-4.0 db4.0 db-4 db40 db4 db-3.3 db3.3 db33 db-3.2 db3.2 db32 db-3.1 db3.1 db31 db-3 db30 db3 db
           do
             AC_CHECK_LIB($dbname, db_create, BDB_LIBADD="$BDB_LIBADD -l$dbname";
               dblib="berkeley"; break, dblib="no")
